.github/workflows/test.yml

```diff
@@ -30,11 +30,20 @@ jobs:
         git clone https://github.com/nvim-treesitter/nvim-treesitter.git
         mv nvim-treesitter $HOME/.local/share/nvim/lazy/
 
+    - name: Install Stylua
+      run: |
+        sudo apt-get install npm -y --no-install-recommends
+        sudo npm install -g @johnnymorganz/stylua-bin
+        make check-fmt
+
     - name: Run luacheck type check / linter
       run: |
         sudo apt-get install lua-check -y --no-install-recommends
         make check
 
+    - name: Check formatting
```

Comment:
don't need this second block

Reply:
Oh I meant to separate the "Install Stylua" stage from the "Check formatting" stage, but I accidently left the `make check-fmt` call in the first block. Do you want these two put together? I thought they may be better separated so that it would be easier to debug if it is the stylua installation that fails.